src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -10,12 +13,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.net.*;
-import java.time.ZoneId;
-import java.time.format.DateTimeFormatter;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
We tend to use specific imports rather than wildcard. Why? At a glance you can read the imports and understand what you class is using. When using a new class with the same name you get a clash of naming which takes time to understand given wildcard is used. http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

You can set intellij to avoid doing that in the settings.